src/langchain_google_memorystore_redis/chat_message_history.py

```diff
@@ -46,15 +42,18 @@ def __init__(
         self._redis = client
         self._key = session_id
         self._ttl = ttl
+        self._encoder = client.connection_pool.get_encoder()
 
     @property
-    def messages(self) -> List[BaseMessage]:
+    def messages(self) -> List[BaseMessage]:  # type: ignore
         """Retrieve all messages chronologically stored in this session."""
         all_elements = self._redis.lrange(self._key, 0, -1)
-        messages = messages_from_dict(
-            [json.loads(e.decode("utf-8")) for e in all_elements]
+
+        assert isinstance(all_elements, list)
```

Comment:
Use if statement not assert

Reply:
+@craig08 

BTW, @averikitsch, I will close this PR next as the code has been merged into the staging branch in another PR. Please let me know if you would like us to raise a PR against main. Thanks!

Reply:
Can you add me as a assignee on the other PRs. Lefts some comments below. Thanks!

Reply:
on a second thought I feel that "assert" is proper here since we are enforcing a contract that shouldn't be broken or there is a serious bug in the redis-py package. I would suggest keeping it.